terminus/generators/monolane_generator.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import print_function
+
+from collections import OrderedDict
+from math import atan2
+from math import degrees
+import yaml
+
+from .city_visitor import CityVisitor
+from models.intersection_node import IntersectionNode
```

Comment:
`IntersectionNode` was removed and now we have `RoadIntersectionNode` for roads and `LaneIntersectionNode` (but based on the code I guess you want `RoadIntersectionNode`)